phpstan.neon

```diff
@@ -1,5 +1,6 @@
 parameters:
     level: 4
+    reportUnmatchedIgnoredErrors: false
```

Comment:
You can ignore it like in the pimcore repository:
```suggestion
    ignoreErrors:
        - '~^Method Doctrine\\DBAL(\\.*)?Connection::query\(\) invoked with \d+ parameters?, 0 required\.\z~'
```

Reply:
Why does this not reported in the lowest dependency... Can you increase the phpstan version in the composer? So both tests are with the newest/equal phpstan version?

Reply:
Maybe we can use executeQuery() instead of query(). query() is deprecated in dbal:
https://github.com/doctrine/dbal/blob/dc9b3c3c8592c935a6e590441f9abc0f9eba335b/lib/Doctrine/DBAL/Connection.php#L1431

Reply:
I substituted query() with executyQuery() and removed the error warning pattern, thanks for the hint.